app/controllers/press_catalog_controller.rb

```diff
@@ -52,24 +52,44 @@ def open_monographs
-        blacklight_config.add_facet_field Solrizer.solr_name('subject', :facetable), label: "Subject", limit: 10, url_method: :facet_url_helper, sort: sort
-        blacklight_config.add_facet_field Solrizer.solr_name('creator', :facetable), label: "Author", limit: 5, url_method: :facet_url_helper, sort: sort
-        if @press.subdomain == 'heb'
-          blacklight_config.add_facet_field Solrizer.solr_name('publisher', :facetable), label: "Publisher", limit: 5, url_method: :facet_url_helper, sort: sort
+      if @press.subdomain == Services.score_press
+        musical_score
+      else
+        if open_monographs >= 15
+          # per page
+          blacklight_config.default_per_page = 15
+          blacklight_config.per_page = [10, 15, 50, 100]
+
+          # facets
+          # Sort HEB facets alphabetically, others by count
+          sort = (@press.subdomain == 'heb') ? 'index' : 'count'
+          blacklight_config.add_facet_field Solrizer.solr_name('open_access', :facetable), label: "Open Access", limit: 1, url_method: :facet_url_helper, sort: sort
+          blacklight_config.add_facet_field Solrizer.solr_name('subject', :facetable), label: "Subject", limit: 10, url_method: :facet_url_helper, sort: sort
+          blacklight_config.add_facet_field Solrizer.solr_name('creator', :facetable), label: "Author", limit: 5, url_method: :facet_url_helper, sort: sort
+          if @press.subdomain == 'heb'
+            blacklight_config.add_facet_field Solrizer.solr_name('publisher', :facetable), label: "Publisher", limit: 5, url_method: :facet_url_helper, sort: sort
+          end
+          blacklight_config.add_facet_field Solrizer.solr_name('series', :facetable), label: "Series", limit: 5, url_method: :facet_url_helper, sort: sort
+          blacklight_config.add_facet_fields_to_solr_request!
         end
-        blacklight_config.add_facet_field Solrizer.solr_name('series', :facetable), label: "Series", limit: 5, url_method: :facet_url_helper, sort: sort
-        blacklight_config.add_facet_fields_to_solr_request!
+
+        search_or_browse
+        monograph_sort_fields
       end
+    end
 
+    def musical_score
+      blacklight_config.add_facet_field Solrizer.solr_name('creator', :facetable), label: 'Composer'
+      blacklight_config.add_facet_field Solrizer.solr_name('octave_compass', :facetable), label: 'Octave Compass'
+      blacklight_config.add_facet_field Solrizer.solr_name('bass_bells_required', :facetable), label: 'Bass Bells Required'
+      blacklight_config.add_facet_field Solrizer.solr_name('solo', :facetable), label: 'Solo'
+      blacklight_config.add_facet_field Solrizer.solr_name('musical_presentation', :facetable), label: 'Musical Presentation'
+      blacklight_config.add_facet_field Solrizer.solr_name('composer_gender', :facetable), label: 'Composer Gender'
```

Comment:
composer_gender and composer_ethnicity should be combined into the single composer_diversity (as they are in the model, presenter and form)

Reply:
Next iteration this will be fixed